packages/next-rest-framework/src/generate/utils.ts

```diff
@@ -227,6 +236,17 @@ export const findConfig = async ({ configPath }: { configPath?: string }) => {
   return config;
 };
 
+async function dynamicallyImportRoute(filePathToRoute: string) {
```

Comment:
This was needed to work with `ts-node`. Importing using the `file://` path wasn't resolving the module correctly. Open to suggestion here as well.